src/stylesheets/components/blog/_article.scss

```diff
@@ -0,0 +1,152 @@
+.article-inner__titleCard {
+  background-color: white;
+  margin: 0 auto rem(58) auto;
+  margin-bottom: rem(58);
+  max-width: 1200px;
+  padding-top: rem(23);
+  width: 100%;
+
+  h1 {
+    font-weight: 300;
+    margin: 0 auto;
+    margin-bottom: 3rem;
+    max-width: 500px;
+    text-align: center;
+    text-transform: uppercase;
+    @include fz(56);
+
+    @include breakpoint(max-width 600px) {
+      @include fz(46);
+    }
+  }
+}
+
+.article-inner__authormeta {
+  border-bottom: 1px solid lighten($c-text, 60%);
+  border-top: 1px solid lighten($c-text, 60%);
+  padding-bottom: rem(19);
+  padding-top: rem(24);
+  text-align: center;
+
+  a {
+    color: $c-purple;
+    display: block;
+    margin-bottom: .25rem;
+    text-decoration: none;
+
+    &:hover, &:focus {
+      text-decoration: underline;
+    }
+  }
+}
+
+.article-container {
+  max-width: 100%;
+  width: 100%;
+
+  header {
+    background: linear-gradient(rgba(196, 165, 0, .7), rgba(196, 165, 0, .7)),
+      url("/images/alice-in-wonderland-big.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    padding-bottom: rem(630);
+    position: relative;
+  }
+}
+.article-header__inner {
+  padding: 1rem;
+
+  img {
+    display: block;
+    margin: 0 auto;
+    max-width: 100%;
+  }
+}
+.article-header__bg {
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 100%;
+  position: absolute;
+  width: rd(572, 1440);
+  z-index: 20;
+}
+
+.article-inner {
+  margin: 0 auto;
+  max-width: 1200px;
+  position: relative;
+  top: -100px;
+  width: rd(950, 1440);
```

Comment:
The max-width is a cover all solution. So for screens 1440px and lower, the percentage of the screen of this block will take up 950px (69.5%) of the screen. But if the screen size is larger the max that percentage can expand is up to 1200px.